Dockerfile.assets

```diff
@@ -1,12 +1,12 @@
 ARG image_tag=latest
 FROM elifesciences/pattern-library_assets-builder:${image_tag} AS assets-builder
-
 FROM tianon/true:latest
-ENV PROJECT_FOLDER=/srv/pattern-library
```

Comment:
`PROJECT_FOLDER` is sometimes read by pipelines to determine where to find the smoke tests. This image should not be affected https://github.com/elifesciences/elife-jenkins-workflow-libs/search?q=PROJECT_FOLDER&unscoped_q=PROJECT_FOLDER